web/modules/custom/bnf/tests/src/Unit/Mapper/ParagraphTextBodyMapperTest.php

```diff
@@ -34,21 +34,22 @@ protected function getEntityClass(): string {
   public function testParagraphTextBodyMapping(): void {
     $this->storageProphecy->create([
       'type' => 'text_body',
+      'field_body' => [
+        'value' => 'This is the text',
+        'format' => 'with_format',
+      ],
     ])->willReturn($this->entityProphecy);
 
     $mapper = new ParagraphTextBodyMapper([], '', [], $this->entityManagerProphecy->reveal());
 
-    $graphqlArticle = ParagraphTextBody::make(
+    $graphqlElement = ParagraphTextBody::make(
+      'textBody',
       Text::make('This is the text', 'with_format')
     );
 
-    $paragraph = $mapper->map($graphqlArticle);
+    $paragraph = $mapper->map($graphqlElement);
 
     $this->assertSame($paragraph, $this->entityProphecy->reveal());
-    $this->entityProphecy->set('field_body', [
```

Comment:
This assertion has been removed, as I place the code directly in the  `create` method, rather than dedicated `set`'s in the mapper.